### PR TITLE
MAINT: remove redundant imports

### PR DESCRIPTION
**Pull request by ceh**

### CODE REVIEW

File: numpy/distutils/mingw32ccompiler.py [hunk 1]
```diff
@@ -71,7 +71,6 @@ def __init__ (self,
         # we need to support 3.2 which doesn't match the standard
         # get_versions methods regex
         if self.gcc_version is None:
-            import re
```
Comment (ceh) [inline numpy/distutils/mingw32ccompiler.py]: Imported on line 15: https://github.com/numpy/numpy/blob/b88a6785b44844b0b4054d9c77aa4a7d7b6bb1ff/numpy/distutils/mingw32ccompiler.py#L15

File: numpy/distutils/misc_util.py [hunk 2]
```diff
@@ -2009,7 +2009,6 @@ def generate_svn_version_py():
                     f.write('version = %r\n' % (version))
                     f.close()
 
-                import atexit
```
Comment (ceh) [inline numpy/distutils/misc_util.py]: Imported on line 8: https://github.com/numpy/numpy/blob/b88a6785b44844b0b4054d9c77aa4a7d7b6bb1ff/numpy/distutils/misc_util.py#L8

File: numpy/lib/utils.py [hunk 3]
```diff
@@ -80,7 +80,6 @@ def __call__(self, func, *args, **kwargs):
         new_name = self.new_name
         message = self.message
 
-        import warnings
```
Comment (ceh) [inline numpy/lib/utils.py]: Imported on line 7: https://github.com/numpy/numpy/blob/b88a6785b44844b0b4054d9c77aa4a7d7b6bb1ff/numpy/lib/utils.py#L7

**Review by jeffyancey — APPROVED** [review]
Looks right to me

**Comment by charris** [comment]
Thanks @ceh .

**Pull request merged**